socorro/processor/mozilla_processor_2015.py

```diff
@@ -57,6 +57,7 @@
         "socorro.processor.mozilla_transform_rules.ThemePrettyNameRule, "
         "socorro.processor.rules.memory_report_extraction"
         ".MemoryReportExtraction, "
+        "socorro.processor.rules.addons_rewrite.AddonsRewriteRule, "
```

Comment:
One of the things the new processor is going to have is better names. "AddonsRewriteRule" is so droll. The new processor is going to have "AddonsRewriteRuleOfWonderAndMystery" or maybe "FrancoisTheAddonsRewriteRule".